RELEASE.md

```diff
@@ -1,5 +1,10 @@
 # Release notes
 
+## 1.0.4
+
+- Adds doc table of contents to root README.md and docs/index.md
+- Renames docs/index.md to [docs/README.md](/docs/README.md)
+
```

Comment:
I couldn't comment directly since the line was not changed by this PR, but don't forget to also change line 29 in this file:

```diff
- - Comprehensive [app documentation](/documentation/index.md) from the ArcGIS for Developers site.
+ - Comprehensive [app documentation](/documentation/README.md) from the ArcGIS for Developers site.
```